templates/monorepo/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "./tsconfig.build.json",
+  "include": ["packages", "types", "scripts", "example"],
+  "compilerOptions": {
+    "allowJs": false,
+    "baseUrl": ".",
+    "typeRoots": ["./node_modules/@types", "./types"],
+    "paths": {
+      "@mono/utils": ["packages/eval/src"],
+      "@mono/react": ["packages/react/src"],
+      "$test/*": ["test/*"]
+    }
```

Comment:
I think it helps with vscode / ts to resolve clicking on types to the src folders of the package instead of dist. I could be wrong and talking out of my ass here, will have to test via trial and error.